### ISSUE TITLE
any future plans for supporting new camera api (apilevel 21)

### ISSUE THREAD

**Issue by slayer35** [boxme/SquareCamera#25]
Hello
in 21  ,google forces to use android.hardware.camera2  api , Do you have any plans for supporting both api level 21 and old version together?
thanks.

**Comment by philipgiuliani** [comment]
No its not forcing. Its just a warning in Android Studio. If you use camera2 API, you can't run the app in API 20.. So there is no other way currently except you want to support API 21 only [(which means about 18% of the devices)..](https://developer.android.com/about/dashboards/index.html)

**Comment by Glurt** [comment]
Could you not use camera2 if the device is running API 21+ and revert to camera1 when its API 19 and below? 

I haven't looked at the docs for either so I'm not sure how difficult this would be but it's certainly an option.

**Comment by philipgiuliani** [comment]
Yes, you could write the camera 2 times (maybe reuse layouts) and do an api check then. What i've heard the apis are completly different.

**Comment by boxme** [comment]
I do have plans to include the new APIs in the coming months
